README.md

```diff
@@ -123,6 +123,8 @@ class LoginFormContainer extends React.Component {
 ## Optional Props
 ### Validation Help
 Used to customize bloom-form's built-in validation. See 'Validation' below.
+### Ignore Focus On First Element
+By default the first element will be focused on.  Set this prop to false to offset default behavior
```

Comment:
Let's say `the first *input*` instead and `...for accessibility reasons.`

Reply:
b075c54